akka-samples/akka-sample-persistence-java-lambda/tutorial/index.html

```diff
@@ -9,7 +9,7 @@
 <h2>Akka Persistence Samples</h2>
 <p>
 This tutorial contains examples that illustrate a subset of
-<a href="http://doc.akka.io/docs/akka/2.4-SNAPSHOT/java/lambda-persistence.html" target="_blank">Akka Persistence</a> features.
+<a href="http://doc.akka.io/docs/akka/2.4.0-RC2/java/lambda-persistence.html" target="_blank">Akka Persistence</a> features.
```

Comment:
these should remain `2.4-SNAPSHOT`, we replace them automatically during the build.